app/src/main/AndroidManifest.xml

```diff
@@ -21,10 +21,42 @@
             android:theme="@style/Theme.PersonalizedAppIconAndroid"
             >
             <intent-filter>
-                <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <!-- Default icon -->
+        <activity-alias
+            android:name=".MainActivityAlias1"
+            android:targetActivity=".MainActivity"
```

Comment:
Not sure whether Android app links will still be able to launch the app when the alias has changed.